src/services/storage/drivers/relational/index.ts

```diff
@@ -65,7 +65,7 @@ export class RelationalStorage implements StorageDriver {
 	url = new (class RelationalUrlStorage {
 		constructor(public storage: RelationalStorage) {}
 
-		public async get(id: string, options = { withInfo: false }): Promise<StoredUrl | UrlWithInformation> {
+		public async get(id: string, options = { withInfo: false }): Promise<StoredUrl | UrlInformation> {
```

Comment:
That's actually a UrlWithInformation.

the variable named urlInfo is the incorrect name here (it should be storedUrlWithInfo).

The story here is that the get function can return only the url, or the url with the info (for admins)